### PR TITLE
Adds MBF DApp Directory API Documentation

### PR DESCRIPTION
**Pull request by themacexpert**
### Description

This PR:

- Adds information about the MBF DApp Directory API
- Adds missing category and tag information from the existing page

### Checklist

- [x] I have added a label to this PR 🏷️
- [x] I have run my changes through Grammarly
- [ ] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [ ] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects
- [ ] If pages have been moved around, I have run the `move-pages.py` script to move the pages and update the image paths on the chinese repo
    - [ ] After the script has been run, I have created an additional PR in `moonbeam-docs-cn`
- [ ] If images have been added, I have run the `compress-images.py` script to compress the images.
- [ ] If variables (in variables.yml) need to be updated (such as a name change), I have updated the `moonbeam-docs-cn` repo to use the new variables
- [ ] If this page requires a disclaimer, I have added one

### Corresponding PRs

Please link to any corresponding PRs here.

### After Translation Requirements

- [ ] Will need to create PR in `moonbeam-docs` repo to remove images
- [ ] Will need to create PR in `moonbeam-docs` repo to remove variables
- [ ] Will need to create PR in `moonbeam-mkdocs` repo to add redirects for Chinese site
- [ ] No additional PRs are required after the translations are done

#### Items to be Updated

Please list any of the items that will need to be added or deleted after the translations are done here.

### CODE REVIEW

**Review by dawnkelly09 — APPROVED** [review]
lgtm. Thanks for updating these!

File: .snippets/code/learn/dapps-list/dapp-directory/nfts.json [hunk 1]
```diff
@@ -0,0 +1,2118 @@
+{
+  "projects": [
```
Comment (eshaben) [inline .snippets/code/learn/dapps-list/dapp-directory/nfts.json:2]: JSON files should use 4 spaces. More info here: https://github.com/papermoonio/documentation-style-guide/blob/main/style-guide.md#code-formatting-by-language

File: .snippets/code/learn/dapps-list/dapp-directory/stellaswap.json [hunk 2]
```diff
@@ -0,0 +1,456 @@
+{
+  "project": {
```
Comment (eshaben) [inline .snippets/code/learn/dapps-list/dapp-directory/stellaswap.json:2]: same comment here - 4 spaces

**Review by eshaben — COMMENTED** [review]
Looks great! Just a minor comment about the formatting of JSON files 🙂

File: .snippets/code/learn/dapps-list/dapp-directory/nfts.json (hunk 1 above)
Comment (eshaben) [reply .snippets/code/learn/dapps-list/dapp-directory/nfts.json:2]: This still isn't using 4 spaces for the indents

File: .snippets/code/learn/dapps-list/dapp-directory/project-data-file.json [hunk 3]
```diff
@@ -1,107 +1,107 @@
...
-            "height": 400,
-            "mimeType": "image/jpeg"
-        },
-        "full": {
-            "fileName": "moonwell-logo-full.jpeg",
-            "width": 3000,
-            "height": 3000,
-            "mimeType": "image/jpeg"
-        }
+  "id": "moonwell",
```
Comment (eshaben) [inline .snippets/code/learn/dapps-list/dapp-directory/project-data-file.json:2]: This was using 4 spaces before? Needs to be updated to use 4 spaces again

File: .snippets/code/learn/dapps-list/dapp-directory/stellaswap.json (hunk 2 above)
Comment (eshaben) [reply .snippets/code/learn/dapps-list/dapp-directory/stellaswap.json:2]: this comment still applies these are only indented using 2 spaces not 4

**Review by eshaben — COMMENTED** [review]
Spacing is still a problem

**Comment by themacexpert** [comment]
fixed, I ran them all thru prettier before thinking that would do it.

**Review by eshaben — APPROVED** [review]
Looks good! ✨

**Pull request merged**